draft-issuance.md

```diff
@@ -0,0 +1,169 @@
+
+# Abstract
+
+This ZIP proposes a change to how nodes calculate the block subsidy.
+
+Instead of following a step function around the four-year halving cycle inherited
+from Bitcoin, we propose a slow exponential “smoothing” of the curve. The new issuance
+scheme would approximate the current 4 year cycle, and results in the last
+zatoshi being spent in around 113 years.
+
+# Motivation
+
+Zcash’s economic model is inherited from Bitcoin and includes the concept of a halving
+mechanism to regulate the issuance of new coins. This approach, though foundational, invites
+a reevaluation amid Zcash’s ongoing evolution. As the network matures, the need to address
+potential future challenges and ensure a sustained and stable economic ecosystem becomes
+apparent. The transition to a smoothed emissions curve offers an opportunity to adjust the network's
+issuance dynamics while maintaining the supply cap of 21,000,000 coins. By doing so, Zcash
+endeavors to optimize its economic framework, accommodating changing circumstances while
+maintaining predictability and stability in rewards distribution.
+
+This proposal outlines a solution to address challenges associated with the existing block
+subsidy issuance mechanism in the Zcash network. The primary goal of this proposal is to
+introduce a more predictable and stable issuance of ZEC by smoothing out the issuance
+curve while preserving the supply cap. It's important to note that this proposal does
+not seek to alter the fundamental aspects of Zcash's issuance policy. The average block
+subsidy size over time will remain the same and the funds for block subsidies will last
+a similar amount of time. Instead, it focuses solely on enhancing the predictability
+and consistency of the block subsidy issuance process.
+
+Smoothing the emissions curve helps ensure that the network remains economically
+viable and stable as it transitions from a traditional issuance mechanism to one
+that maintains a sustainable and predictable issuance of rewards over time. It
+prevents abrupt changes in the rate of newly issued coins, which could lead to
+disruptions in the network's economic model and potentially impact its security
+and sustainability. A smoother emissions curve allows for a more gradual and controlled
+transition, providing ZEC stakeholders and participants with a clear understanding of
+how rewards will be distributed over time.
+
+
+
+# Specification
+
+Smoothing the issuance curve is possible using an exponential decay formula that
+satisfies the following requirements:
+
+## Requirements
+
+1. Block subsidies MUST be weakly decreasing
+2. Block subsidies SHOULD approximate a continuous function
+3. When `AVAILABLE_SUBSIDIES(h) > 0` then block subsidies for block `h`
+MUST always be `> 0`, preventing a final “unmined” zatoshi
+4. For any 4 year period, all paid out block subsidies MUST equal approximately
+half of `AVAILABLE_SUBSIDIES` at the beginning of that 4 year period
```

Comment:
This suggestion needs to be re-applied, because the current text contains an off-by-one error. It also references a definition I suggested removing earlier:
https://github.com/zcash/zips/pull/706/files#r1330645529